src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -138,6 +140,9 @@
             wait-time]
      :or   {format    :transit
             wait-time 0}}]
+   (trouble/debug-troublemaker :receive-message
```

Comment:
let me know if I'm reading this correctly. So you're keeping an atom in the troublemaker ns, and this here reads the atom to see if a flag is set. And if a flag is set, then you raise the given exception. The only reason for this being here is for testing?

If that's all true, then this is a bad code smell. We should never hide test code in production code with hacks.